### DISCUSSION TITLE
Pet-Shop example running npm run dev does not open browser window

### DISCUSSION THREAD

**Discussion by lsman333** [ConsenSys-archive/truffle#5842] [General Q&A]
Hi all,
I have following all the steps of the pet shop tutorial, and so far no errors, but when I am at the stage of executing npm run dev, I don't get any browser tab opening up to show me the dapp.  
My terminal output shows

> > pet-shop@1.0.0 dev
> > lite-server
>
>** browser-sync config **
>{
>  injectChanges: false,
>  files: [ './**/*.{html,htm,css,js}' ],
>  watchOptions: { ignored: 'node_modules' },
>  server: {
>    baseDir: [ './src', './build/contracts' ],
>    middleware: [ [Function (anonymous)], [Function (anonymous)] ]
>  }
>}

and that's it... no further output until I press Ctrl-C.
Any ideas?

**Comment by FlaviuWeb3** [comment]
Hi @lsman333 could you please push your code into a public repository and share the link with me in order to reproduce it?

**Comment by lsman333** [comment]
Oddly enough, after waiting enough time, the browser tab eventually opened... I have no idea what triggered this.  I think it took several hours.  And every time that I run npm run dev, I again have to wait some random and long amount of time before it opens the tab.  
This is on a 2020 Macbook Air M1 btw.
Eventually the pet shop loaded, and clicking on Adopt, MetaMask opens and asks to approve the transaction... Ganache updates with the transaction and gas fee appears to be debited.  However, the Adopt button never changes state even after refreshing.
